maven-enforcer-rules/README.md

```diff
@@ -33,11 +33,11 @@ Here is an example pom.xml:
                             <helidonDependenciesRule>
                                 <namespace>JAKARTA | JAVAX</namespace>
                                  <!-- list of strings - can be used to exclude a package / group name from validation -->
-                                 <excludedGavRegExs>
-                                    <!-- for example only - we suggest not including this exclusion in your usage -->
-                                    <excludedGavRegEx>javax.servlet.*</excludedGavRegEx>
+<!--                                 <excludedGavRegExs>-->
```

Comment:
It was fine as it was since it's the example snippet in the README.

Reply:
I find that people are just copying it and using it as-is.